lib/internal/inspector/_inspect.js

```diff
@@ -44,88 +74,72 @@ class StartupError extends Error {
   }
 }
 
-function portIsFree(host, port, timeout = 9999) {
-  if (port === 0) return Promise.resolve(); // Binding to a random port.
+async function portIsFree(host, port, timeout = 9999) {
+  if (port === 0) return PromiseResolve(); // Binding to a random port.
 
   const retryDelay = 150;
-  let didTimeOut = false;
+  const ac = new AbortController();
```

Comment:
Can we import this from the internal module (unless you're supposed to be using the global) ?